lib/team.rb

```diff
@@ -0,0 +1,19 @@
+class Team
+
+  attr_reader :team_id,
+              :franchise_id,
+              :team_name,
+              :abbreviation,
+              :stadium,
+              :link
+
+  def initialize(details)
+    @team_id = details[:team_id].to_i
+    @franchise_id = details[:franchiseId].to_i
```

Comment:
team and franchise id numbers changed to integers. all others remain as default strings. this can be changed later if we would rather work with strings for ids.

Reply:
yeah same with the game class we can make a decision late